apps/storybook-react/stories/Avatar.stories.tsx

```diff
@@ -20,10 +20,13 @@ const Wrapper = styled.div`
 export default {
   title: 'Components/Avatar',
   component: Avatar,
-  decorators: [withKnobs],
-}
+} as Meta
 
-export const Examples = () => (
+export const Default: Story<AvatarProps> = (args) => (
+  <Avatar src={image} {...args}></Avatar>
```

Comment:
Shouldn’t this be self-closing?

```suggestion
  <Avatar src={image} {...args} />
```

Reply:
Yes! I'll fix!